### ISSUE TITLE
Web server should listen on 0.0.0.0 (all addresses) or be configurable

### ISSUE THREAD

**Issue by Tawmu** [rnwood/smtp4dev#22]
At the moment it seems like the web service in smtp4dev 3.0 alpha only listens on 'localhost', which works fine if you're developing locally but not so well if you're relying on remote machines. It would be very useful if the listening address was configurable in the config file, similar to how the listening port is.

**Comment by lennartvdd** [comment]
Its possible by setting the environment variable ASPNETCORE_URLS before calling the executable 
(or the noruntime version via `dotnet Rnwood.Smtp4dev.dll`)

```
# Unix:
ASPNETCORE_URLS="http://0.0.0.0:5000" ./Rnwood.Smtp4dev

# Windows PowerShell:
$env:ASPNETCORE_URLS="http://0.0.0.0:5000" ; .\Rnwood.Smtp4dev.exe

# Windows CMD (note: no quotes):
SET ASPNETCORE_URLS=http://0.0.0.0:5000 && Rnwood.Smtp4dev.exe
```
Found the solution here: https://github.com/aspnet/KestrelHttpServer/issues/639#issuecomment-237721352

**Comment by rnwood** [comment]
Resolved for now by merging #26. 
Hope to revisit this and make it much easier to discover when exposing the options in the UI, but for now have called out how to do this in the README.
